frontend/src/components/activePage/TableWorkspaceLogic/TableWorkspaceLogic.tsx

```diff
@@ -36,178 +32,101 @@ export interface ITableWorkspaceLogicProps {
 }
 
 const TableWorkspaceLogic: FC<ITableWorkspaceLogicProps> = ({ ...props }) => {
-  const { workspaces, userId, tenantNamespace } = props;
-  const [templatesMapped, setTemplatesMapped] = useState<Array<Template>>();
-  //const [dataTemplate, setDataTemplate] = useState<WorkspaceTemplatesQuery>();
+  const { workspaces, userId, tenantNamespace, filter } = props;
   const [
     dataInstances,
     setDataInstances,
   ] = useState<InstancesLabelSelectorQuery>();
 
+  const label = `crownlabs.polito.it/workspace in (${workspaces.map(
+    ({ id }) => id
+  )})`;
```

Comment:
```suggestion
  const label = `crownlabs.polito.it/workspace in (${workspaces.map(
    ({ id }) => id
  )}).join(',')`;
```
Isn't something alike needed? 🤔

Reply:
`crownlabs.polito.it/workspace in (sid,ttpu-os-2021) TableWorkspaceLogic.tsx:44`
this is the value of 'label' in my case... it seems work

Reply:
I'd still make it explicit, also to ease understanding the output. (mind that there's an issue with parentheses at the end in my suggestion)